integration

```diff
@@ -1375,5 +1375,6 @@
   "zhbjsh/homeassistant-ssh",
   "zigul/HomeAssistant-CEZdistribuce",
   "ZsBT/hass-w1000-portal",
-  "zubir2k/homeassistant-esolatgps"
+  "zubir2k/homeassistant-esolatgps",
+  "zubir2k/homeassistant-esolattakwim"
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>